src/App.jsx

```diff
@@ -1,14 +1,21 @@
 import React from 'react';
 import './App.css';
+import { Sum } from './components/Sum/Sum';
 
 export const App = () => (
   <>
-    <p>Sum of 2 and 3 is 5</p>
-    <p>Sum of -5 and 5 is 0</p>
-    <p>Sum of 10 and 0 is 10</p>
-    <p>Sum of 0 and 5 is 5</p>
-    <p>Sum of 0 and 0 is 0</p>
-    {/* Replace paragraphs with Sum componets */}
-    {/* And remove commented lines :) */}
+    <Sum
+      a={2}
+      b={3}
+    />
```

Comment:
```suggestion
    <Sum a={2} b={3} />
```
![image](https://github.com/mate-academy/react_sum/assets/96409563/baccb1cb-2fee-406e-bf36-151cd3cab67e)